projects/extension/Dockerfile

```diff
@@ -1,11 +1,31 @@
 # syntax=docker/dockerfile:1.3-labs
-ARG PG_MAJOR
+ARG PG_MAJOR=16
 ARG TIMESCALEDB_IMAGE=timescale/timescaledb-ha:pg${PG_MAJOR}
 FROM ${TIMESCALEDB_IMAGE} as timescaledb
 
-ARG PG_MAJOR
-FROM postgres:${PG_MAJOR}
+FROM postgres:${PG_MAJOR} AS build-latest
+ENV PG_MAJOR=${PG_MAJOR}
+ENV PIP_BREAK_SYSTEM_PACKAGES=1
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends make python3-pip git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+WORKDIR /pgai
+COPY requirements.txt .
+COPY ai/__init__.py ai/
+RUN version=$(grep -oP '(?<=__version__ = ")[^"]*' ai/__init__.py) && \
+    mkdir -p /usr/local/lib/pgai/$version && \
+    pip3 install -v --no-deps --compile -t /usr/local/lib/pgai/$version -r requirements.txt
```

Comment:
why are we doing this? `make install` does this already

Reply:
To cache de dependencies in a Docker layer. If we change the extension code we don't need to re-install the dependencies.

Reply:
ah. yeah. bummer